Update 2101 UG to include Utility commands and final section (#285)
AY2324S1-CS2103T-F11-1/tp: docs/2101UserGuide.md

<hr class="feature-class-separator">

### Utility commands
CCACommander provides you with powerful utility commands to make your admin tasks even more of a breeze than it was.

#### Undoing a command: `undo`

The `undo` command is the ultimate safety net for CCACommander. You no longer have to fear accidentally deleting 
the wrong event or enrolling the wrong member. Simply enter the `undo` command to undo your previous command, 

> and it will revert to the state it was before.

Reviewer: do you think this line (307) sounds abit ambiguous but idk how to rephrase too so i guess no change is okay also :')

Reviewer 2: Can consider:

Simply enter the `undo` command to revert back to its previous state.

Author: I will change it to `return to the previous state`

#### Redoing a command: `redo`

> Accidentally used `undo`? Don’t worry, you can redo your undone commands to retrieve your previous data too!

Reviewer 2: Same nit as below, can consider "recover" instead of "retrieve"
```suggestion
Accidentally used `undo`? Don’t worry, you can redo your undone commands to recover your previous data too!
```

Or simply omit it entirely:
```suggestion
Accidentally used `undo`? Don’t worry, you can redo your undone commands too!
```

Author: I will be omitting it

Here’s how you can use it: `redo`

Here's a list of commands you can redo:
* `clear`
* `createMember`
* `deleteMember`
* `editMember`
* `createEvent`
* `deleteEvent`
* `editEvent`
* `enrol`
* `unenrol`

> * `editEnrolment`

Reviewer 2: I think the list of commands here is not applicable to the `redo` command, since you can only redo a command that was undone. In other words, you cannot specifically redo a `clear` or `enrol` command, you can only redo an `undo` command.

Author: good point I'll remove this list

<hr class="command-separator">

#### Clear all Members and Events: `clear`
Is it time for a new academic year when you need to remove all your previous members and events for new members and 
events? You can use the `clear` command to instantly remove all your members and events without needing to delete them one by one.

Here’s how you can use it: `clear`

<div markdown="block" class="alert alert-primary">:bulb: Accidentally cleared your list? 

> Worry not as you can use the `undo` command to retrieve your data! 

Reviewer 3: ```suggestion
Worry not as you can use the `undo` command to recover your data! 
```
Does recover sound better?

Reviewer: yes agree 
![image](https://github.com/AY2324S1-CS2103T-F11-1/tp/assets/99804772/5887a783-9467-417b-9201-c4fb92aec8e4)

Author: yes rceover is good!

</div>

<hr class="command-separator">

#### Help: `help`

> Forgot how to edit a specific enrolment or how to unenrol members? The `help` command will allow you to clarify and recall any features of the app by redirecting you to our friendly and informative user guide.

Reviewer: I think not directly redirecting, so maybe
```suggestion
Forgot how to edit a specific enrolment or how to unenrol members? The `help` command will allow you to clarify and recall any features of the app by providing you with a redirection link to our friendly and informative user guide.
```

Author: I have changed it to `providing you with a link`

**Q**: Do I need Wi-Fi connection to use CCACommander?<br>
**A**: No, you can use CCACommander offline.

**Q**: How much storage space do I need for CCACommander?<br>
**A**: CCACommander is so lightweight that you only need 30 MB on your computer.

**Q**: What happens if my laptop battery dies suddenly?<br>
**A**: Rest assured, your data remains securely stored on your laptop. 
Once your laptop is recharged, you can regain access to all your stored information.

--------------------------------------------------------------------------------------------------------------------

## Known issues

1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
2. **Edit** commands go back to the main view no matter what view you were in.

--------------------------------------------------------------------------------------------------------------------

## Glossary
This glossary provides definitions for the special terms used in this user guide.

> **CCA**: Co-curricular activity

Reviewer: ```suggestion
**CCA**: Co-curricular activity.
```

Author: Done!
